lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -0,0 +1,60 @@
+{
+  "workflows": [
+    {
+      "name": "default",
+      "label": "Digital collections workflow",
+      "description": "A multi-step workflow for digital collections",
+      "actions": [
+         {
+           "name": "finalize_digitization",
```

Comment:
I think we ought to either:
1. create the mediated deposit workflow first
1. don't bother putting the notification service into a workflow.

If we don't pick one of these, we going to have to have another PR to undo this stuff.